app/javascript/packages/document-capture/context/marketing-site.jsx

```diff
@@ -4,10 +4,11 @@ import { createContext } from 'react';
  * @typedef MarketingSiteContext
  *
  * @prop {string} documentCaptureTipsURL Link to Help Center article with tips for document capture.
+ * @prop {string} supportedDocumentsURL Link to Help Center article detailing supported documents.
```

Comment:
would it make sense to do something like `@see AppRootData` or something to minimize duplication?

Reply:
> would it make sense to do something like `@see AppRootData` or something to minimize duplication?

Hm, yeah, I see the concern about the duplication. I think it might be nice if we could set up `AppRootData` to be an intersection type of all the different context values it's picking from, or vice-versa this context type a `Pick<AppRootData, 'foo'|'bar'|'baz'>`. One issue is that the dataset capitalization for the properties is different from the context capitalization, so the names don't line up exactly.

I think it could just be enough to do like you suggest with a `@see`, even if it's not "linked up" in a way that TypeScript/IDE/etc could follow, and avoid the extended property descriptions in all but the one canonical source.